easybuild/tools/config.py

```diff
@@ -79,9 +79,10 @@
     CONT_IMAGE_FORMAT_SQUASHFS,
 ]
 
+CONT_TYPE_APPTAINER = 'apptainer'
 CONT_TYPE_DOCKER = 'docker'
 CONT_TYPE_SINGULARITY = 'singularity'
-CONT_TYPES = [CONT_TYPE_DOCKER, CONT_TYPE_SINGULARITY]
+CONT_TYPES = [CONT_TYPE_DOCKER, CONT_TYPE_SINGULARITY, CONT_TYPE_APPTAINER]
```

Comment:
Why not alphabetical order here? You seem to insist on that in most other places where order is otherwise not important.

Reply:
fixed in f9f2984